open_rfps/Decentralized Forum RFP

```diff
@@ -0,0 +1,65 @@
+
+**Proposal Goals (list with explanations where possible)**
+The main goal is to create a new decentralized forum that will be used by many people and will also bring new users to the Sora network. 
+The forum won't require registration by using email or any kind of KYC, people will be able to post threads and comments 
+by only signing messages with their wallets (polkadot.js). There are various other crypto-forums/platforms (i.e. bitcointalk) that have many million users. 
+Only last year over 20 million users (combined) registered to these forums. 
+There will be various sections in the forum but the very interesting one will be the Marketplace where people will freely trade, 
+buy and sell digital and non-digital goods using a contract system(at their own risk).
+The XOR will be used for all the payments in the forum (like subscriptions, contracts, paid stickers, etc).
+To run the forum, me and my team will have to register a company, so we can pay all the people who will work for the project legally (coders, UI designers, mods, etc.), pay for
+the servers that will host the forum and also probably we will use some of the funds for marketing. I will write the proposed budget below.
+
+**Scope of Work (describe the scope of work in as much detail as possible)**
+All the technology that we may need to code the forum is available. Polkadot.js wallet seems to be compatible with whatever we have in our mind as team. 
+Would be great if we could use wallet connect for Fearless Wallet, Sora DAE and other mobile wallets. This may happen in the future.
+
+**Current roadblocks**
+None 
+
+**Evaluation metrics and criteria**
+- People will be able to post comments and threads by only signing messages with their wallets.
+
+- There will be sections such as Economy, Life, Code, Games, Tech, Science, Market, Cryptocurrencies, and many others.
+
+- To prevent spam posts/threads, it will be required each user to hold at least 40-50$ worth of XOR in their address (the equivalent amount of $ may change).
+
+- There will be an exclusive Sora and Polkaswap section in the forum so all the users can read about new updates and learn more about the network.
+
+- We will create a contract system for the Marketplace section, people will give a score for the quality of the products/services of the vendors after every deal.
+
+- The contracts will be paid in XOR, they'll be off-chain contracts in a server, in the future we may upgrade the contract system and they may be on-chain contracts to the public Sora's blockchain.
+
+- There will be groups owned by influencers/youtubers in order to bring more users, people will pay for subscriptions the same way they pay at other platforms, so they have access to each group.
+
+- Every group will have its section in the forum and only people who paid subscriptions will have access.
+
+- Subscriptions, group sales, contracts, stickers, and in general everything in the forum will be paid only with XOR.
```

Comment:
Where the information about subscriptions, sales, contracts etc will be stored?

Reply:
They will be stored on the server that will host the forum and it will be controlled by the main team but at the same time, the information will be in public for everyone to view it as we will use only Sora Addresses and we will announce what are the official addresses for every group and also for the group sales, paid stickers, and everything.

Reply:
They will be stored on the server that will host the forum and it will be controlled by the main team but at the same time, the information will be in public for everyone to view it as we will use only Sora Addresses and we will announce what are the official addresses for every group and also for the group sales, paid stickers, and everything.
Just because you might misunderstand. I'll try to explain with an example:

Let's say you want to buy 1 month subscription for a group. Every group's Sora address for the payment will be known in public, everyone can see how many people purchased a sub.

What will be stored in the server, is the buyer's address, so when they visit the forum and connect their Sora-Account, they will have access to the group. This is the ''centralized'' part. We couldn't somehow make it decentralized at this stage, I'm sure that in the future we will find a way to make it more decentralized.
So when I say that the info will be stored in the server I mean the info that says ''This address has access to this group in the forum''.